src/components/TokenList/TokenList.tsx

```diff
@@ -184,7 +184,6 @@ const TokenList = ({
               </TokenContainer>
             )}
             {inactiveTokens.length !== 0 &&
-              tokenQuery &&
               sortedFilteredTokens.length < 5 && (
```

Comment:
```suggestion

```

This line should also be removed or else the tokens still won't appear if the `sortedFilteredTokens` are more than 5. All else seems fine!

Reply:
I removed this condition but i have another question: Do we still show to the user only 10 token from the inactive one by default or do we have to increase the list ? Maybe we can increase to 50 - 100, what do you think ? 

![image](https://user-images.githubusercontent.com/9848115/173179884-b9d2a628-61e6-405b-9449-8c98f25ec6b1.png)

Reply:
Not sure, did you ask around in the dev chat yet?